tests/test_network.py

```diff
@@ -1,34 +1,32 @@
+# pylint: : disable-all
 #
 # Copyright (c) 2020-2022, RTE (http://www.rte-france.com)
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 import copy
+import datetime
+import pathlib
 import re
+import tempfile
 import unittest
-import datetime
 from os.path import exists
 
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
 import pandas as pd
+import pypowsybl as pp
+import pypowsybl.report as rp
 import pytest
+import util
 from numpy import NaN
-import numpy as np
-
-import pypowsybl as pp
-import pathlib
-import matplotlib.pyplot as plt
-import networkx as nx
-
 from pypowsybl.network import ValidationLevel
 
-import util
-import tempfile
-
-import pypowsybl.report as rp
 
 TEST_DIR = pathlib.Path(__file__).parent
-DATA_DIR = TEST_DIR.parent / 'data'
+DATA_DIR = TEST_DIR.parent / "data"
```

Comment:
The PR should only contain changes related to your feature, not changes in the whole file.
Also, we need to take care to respect the existing conventions in new code (single quotes instead of double quotes for example here).

Reply:
I tried to make the file consistent to the pylint norm

Reply:
Understood but such a large change should go into another PR to separate the concerns.

About the particular question of quotes, pylint does not dictate anything between the 2, but consistency matters:
in pypowsybl and in many codebases, we have chosen single quote for simple strings (and 3xdouble quotes for docstrings and multi-line strings).

Reply:
ok, i have noted my todos:

- Maintain consistency for names of elements (Easy)
- Look for Java call to make number of elements easily and dynamically updateable (Difficult)
- Use single quotes (ok)